base/shell/cmd/cmd.c

```diff
@@ -2254,6 +2254,22 @@ Initialize(VOID)
             {
                 OutputStreamMode = UTF16Text;
             }
+            else if (option == _T('F'))
+            {
+                if (!_tcsnicmp(&ptr[2], _T(":OFF"), 4))
+                {
+                    /* Disable file and path completion */
+                    AutoCompletionChar = 0x20;
```

Comment:
Is this the best way to disable these features?

Reply:
Yes, just grep for AutoCompletionChar.